lib/routes/students.js

```diff
@@ -147,7 +147,7 @@ module.exports = [
         },
         payload: {
           campus: Joi.string().valid(CONSTANTS.campus),
-          donor: Joi.array().items(Joi.number().integer().valid(CONSTANTS.donor.map((ids)=>ids.id))).min(1),
+          donor: Joi.array().items(Joi.number().integer().valid(CONSTANTS.donor.map((ids)=>ids.id))),
```

Comment:
@kirithiv2000 Have you tested this? I doubt this works without setting it to `.allow(null)`